src/api-guides/e-receipts.markdown

```diff
@@ -1,173 +1,159 @@
+- [Password Grant](/api-reference/authentication/apidoc.html#password_grant) - Connect from the SAP Concur App Center.
+- [Authorization Grant](/api-reference/authentication/apidoc.html#auth_grant) - Connect from the Partner App.
+- [One-Time Password Grant](/api-reference/authentication/apidoc.html#otp_grant) - Connect from the Partner App using email.
 
-**Existing App Center Partners:**  The Concur Platform team will need to configure your existing sandbox among other things to enable access.  You can make that request by contacting your [Alliance Manager](mailto:AppCenterAlliance@concur.com?subject=Receipts%20v4%20Migration%20Request).
+### App Center Flow - Connecting from SAP Concur Web / Mobile App Center
 
-
-### User Connections / Authentication
-
-The below Authentication Methods are available to obtain an access token.  Password Grant or Authorization Grant or One-Time Password Grant is required for Certification:
-
-- [Password Grant](/api-reference/authentication/apidoc.html#password_grant) – Connect from the Concur App Center.
-- [Authorization Grant](/api-reference/authentication/apidoc.html#auth_grant) – Connect from the Partner App.
-- [One-Time Password Grant](/api-reference/authentication/apidoc.html#otp_grant) – Connect from the Partner App utilizing Email.
-
-
-#### App Center Flow - Connecting from Concur Web / Mobile App Center
-
-1. To begin the authentication flow, a request (temporary) token is obtained via the Concur AppCenter interface.
-2. A logged-in Concur user clicks on the Connect button on the App listing in the App Center and authorizes your app to post data.   App Listing Example below:
+1. To begin the authentication flow, a request (temporary) token is obtained via the SAP Concur AppCenter interface.
+2. A logged-in SAP Concur user clicks on the Connect button on the App listing in the App Center and authorizes your app to post data.   App Listing Example below:
 
     ![appflow_listing](/assets/img/api-guides/e-receipts/appflow_listing.png)
 
-3. Concur&#39;s authorization service will redirect the user to your landing page.  Please follow the [App Center Design Guidelines](/manage-apps/go-market-docs/app-center-ux-guidelines-consumer.html) to create a web page that listens for an HTTP GET request from Concur.
-4. The redirect URI will contain an **id** and **requestToken** parameters.  
+3. The SAP Concur authorization service will redirect the user to your landing page. Please follow the [App Center Design Guidelines](/manage-apps/go-market-docs/app-center-ux-guidelines-consumer.html) to create a web page that listens for an HTTP GET request from SAP Concur.
+4. The redirect URI will contain an `id` and `requestToken` parameters.  
 Example:   https://{partner_redirect_URI}?id=8568a4cd-8ffc-49d6-9417-be2d69aa075f&requestToken=5l85ae5a-426f-4d6f-8af4-08648c4b696b
```

Comment:
If we can we should put the example on a separate line with triple ticks:

4. The redirect URI will contain an `id` and `requestToken` parameters. Example:

```
https://{partner_redirect_URI}?id=8568a4cd-8ffc-49d6-9417-be2d69aa075f&requestToken=5l85ae5a-426f-4d6f-8af4-08648c4b696b
```

5. When your application receives the redirect call strip the `id` and `requestToken` values